src/components/pages/Contacts/index.tsx

```diff
@@ -166,13 +190,84 @@ const UnconnectedContacts: React.FC<PropsFromRedux> = ({
 
         {orgContacts.loading && spinner}
 
+        <Modal show={showTagModal}>
+          <p className="m-2">
+            Select the tags you want to apply to all {countContacts} selected
+            contacts:
```

Comment:
Here is the copy i'm thinking about changing - maybe something like "Select the tags you want to apply to all X selected contacts. All other tags on this contact will be deleted:"